Build/transifex/English.Plugins.xlf

```diff
@@ -0,0 +1,1442 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
❗️  There must be no changes under Build/ folder.

Reply:
Fixed.